crates/sui-graphql-rpc/src/types/big_int.rs

```diff
@@ -35,6 +35,13 @@ impl Description for BigInt {
         "String representation of an arbitrary width, possibly signed integer."
     }
 }
+impl BigInt {
+    pub fn to_u64(&self) -> InputValueResult<u64> {
+        self.0
+            .parse()
+            .map_err(|e| InputValueError::custom(format!("Not a valid u64 number: {e}")))
+    }
+}
```

Comment:
Ah, I see that it's for the gas price and gas budget -- if that's all we're using it for, I think it's safe to leave that as `Int`, because although the Max SUI in circulation (denominated in MIST) exceeds 2^53, you can't pay more than 50 SUI for a single transaction.